src/PKSim.Core/Services/CreateIndividualAlgorithm.cs

```diff
@@ -28,7 +28,8 @@ public interface ICreateIndividualAlgorithm
       /// <param name="minWeight">minimal weight allowed for the randomized individual. Null means not constrained</param>
       /// <param name="maxWeight">maximal weight allowed for the randomized individual. Null means not constrained</param>
       /// <param name="allIndividualParameters">All individual parameters defined in the individual (optimization purpose only)</param>
-      void Randomize(Individual individual, IParameter weightParameter, double? minWeight, double? maxWeight, IEnumerable<IParameter> allIndividualParameters);
+      /// <param name="randomGenerator">Random generator used to create values</param>
+      void Randomize(Individual individual, IParameter weightParameter, double? minWeight, double? maxWeight, IEnumerable<IParameter> allIndividualParameters, RandomGenerator randomGenerator);
```

Comment:
Passing the RandomGenerator as parameter is a way to ensure that we can reproduce the same population outputs (comes from the population)